src/launcher/features/settings/settingsSlice.ts

```diff
@@ -9,18 +9,22 @@ import { createSlice } from '@reduxjs/toolkit';
 
 import {
     getCheckForUpdatesAtStartup as getPersistedCheckForUpdatesAtStartup,
+    getIsQuickstartInfoShownBefore as getPersistedIsQuickstartInfoShownBefore,
     setCheckForUpdatesAtStartup as setPersistedCheckForUpdatesAtStartup,
+    setQuickstartInfoWasShown as setPersistedQuickstartInfoWasShown,
 } from '../../../ipc/persistedStore';
 import type { RootState } from '../../store';
 
 export type State = {
     shouldCheckForUpdatesAtStartup: boolean;
     isUpdateCheckCompleteVisible: boolean;
+    isQuickstartInfoShownBefore: boolean;
 };
 
 const initialState: State = {
     shouldCheckForUpdatesAtStartup: getPersistedCheckForUpdatesAtStartup(),
     isUpdateCheckCompleteVisible: false,
+    isQuickstartInfoShownBefore: getPersistedIsQuickstartInfoShownBefore(),
```

Comment:
Maybe rename this and the persisted version to `wasQuickstartInfoShownBefore` for consistency with the reducer and for some reason I keep getting confused as to what is meant when I read it for a second.

Reply:
There is this typical convention to prefix boolean stuff with `is` (and maybe `has`). I don't always like that convention but usually follow it. Because of that I hesitate to prefix this with `was` even though it would sound nicer.